ajax/libs/jquery-nice-select/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "jquery-nice-select",
+  "filename": "jquery.nice-select.min.js",
+  "version": "1.1.0",
+  "description": "A lightweight jQuery plugin that replaces native select elements with customizable dropdowns.",
+  "keywords": [
+    "jquery",
+    "dropdown"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/hernansartorio/jquery-nice-select.git"
+  },
+  "author": {
+    "name": "Hernán Sartorio",
+    "url": "https://github.com/hernansartorio"
+  },
+  "license": "ISC",
```

Comment:
@hare1039 You should try to confirm the license with author like https://github.com/hernansartorio/jquery-nice-select/issues/31 but there is no response. Try to open an issue to discuss it. Thanks.